InfiniteLoopBookProject/src/com/st/lms/dao/DAOBasic.java

```diff
@@ -0,0 +1,22 @@
+package com.st.lms.dao;
+
+import java.io.IOException;
+
+public interface DAOBasic {
+//	public List<Entry> getAllEntries();			//loads all entries from database

```

Comment:
May want to delete these commented out section if you do not plan on using them.